*** snapcast-0.12.0/server/Makefile.org	2017-10-24 13:09:45.978146071 +0200
--- snapcast-0.12.0/server/Makefile	2017-10-24 13:10:25.815149554 +0200
***************
*** 58,63 ****
--- 58,70 ----
  LDFLAGS  += -lavahi-client -lavahi-common -latomic
  OBJ      += ../common/daemon.o publishZeroConf/publishAvahi.o 
  
+ else ifeq ($(TARGET), LIBREELEC)
+ 
+ STRIP     = echo
+ CXXFLAGS += -pthread -DNO_CPP11_STRING -DHAS_TREMOR -DHAS_ALSA -DHAS_AVAHI -DHAS_DAEMON -DHAS_EXPAT
+ LDFLAGS  += -lasound -lvorbis -lavahi-client -lavahi-common -logg -lexpat
+ OBJ      += ../common/daemon.o publishZeroConf/publishAvahi.o
+ 
  else ifeq ($(TARGET), FREEBSD)
  
  CXX       = g++
