*** snapcast-0.12.0/client/Makefile.org	2017-10-24 12:26:25.204852669 +0200
--- snapcast-0.12.0/client/Makefile	2017-10-24 12:27:02.817796902 +0200
***************
*** 59,64 ****
--- 59,71 ----
  LDFLAGS  += -lasound -lvorbisidec -lavahi-client -lavahi-common -latomic
  OBJ      += ../common/daemon.o player/alsaPlayer.o browseZeroConf/browseAvahi.o
  
+ else ifeq ($(TARGET), LIBREELEC)
+ 
+ STRIP     = echo
+ CXXFLAGS += -pthread -DNO_CPP11_STRING -DHAS_TREMOR -DHAS_ALSA -DHAS_AVAHI -DHAS_DAEMON
+ LDFLAGS  += -lasound -lvorbis -lavahi-client -lavahi-common -logg
+ OBJ      += ../common/daemon.o player/alsaPlayer.o browseZeroConf/browseAvahi.o
+ 
  else ifeq ($(TARGET), BUILDROOT)
  
  CXXFLAGS += -pthread -DNO_CPP11_STRING -DHAS_TREMOR -DHAS_ALSA -DHAS_AVAHI -DHAS_DAEMON
